backend/main.py

```
@@ -1,369 +1,407 @@
-from fastapi import FastAPI, HTTPException, Body
-from typing import List
+from fastapi import FastAPI, HTTPException, Body, WebSocket, Request, Depends, Header
+from fastapi.middleware.cors import CORSMiddleware
+from fastapi.middleware.trustedhost import TrustedHostMiddleware
+from fastapi.middleware.httpsredirect import HTTPSRedirectMiddleware
+from typing import List, Dict, Optional
 import uuid
```

Ruff (0.11.9)

`uuid` imported but unused

Remove unused import: `uuid`

(F401)